examples/analog_hamiltonian_simulation/06_Analog_Hamiltonian_simulation_with_PennyLane.ipynb

```diff
@@ -0,0 +1,801 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">H_\text{cost} = \sum_{i=1}^8\left(x_i-\frac{1}{2}\right)\left(x_{i+1}-\frac{1}{2}\right).  \end{align} 
</pre><p> </p><p>Do we start from i = 0 here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yes, good catch. I will make the same change in our blog post.